### DISCUSSION TITLE
How can I stop this tendency of multiple folders

### DISCUSSION THREAD

**Discussion by amowogbaje** [spatie/laravel-medialibrary#3556] [Q&A]
`$media = auth()->user()->addMedia($this->upload)->toMediaCollection('images', 'uploads');`

The code above storing each images I uploaded in a folder named by the media->id value.

How can I stop this , I don't want to touch the code in the /\vendor\spatie\laravel-medialibrary\src\Support\PathGenerator\DefaultPathGenerator.php

Please what can I do 
![Capture](https://github.com/spatie/laravel-medialibrary/assets/8771058/cfdd6e80-ffb8-41a1-bf70-d3539522f742)

**Comment by drnkwtr** [comment]
Create custom path generator. But don't recommend you to remove id's folders cause of library will work wrong without ids
